src/HelpersServiceProvider.php

```diff
@@ -32,6 +33,24 @@ public function boot()
             return !$result;
         });
 
+        Validator::extend('list_exists', function ($attribute, $value, $parameters) {
```

Comment:
let's move both validator extension methods to the separate function `extendValidator`